### ISSUE TITLE
Finish documentation

### ISSUE THREAD

**Issue by paquettg** [paquettg/leaguewrap#60]
I would like to finish the documentation on the README to cover the remaining end points that have little/no documentation. 

This is primary a self note.

**Comment by KioCoan** [comment]
Could the next one be the League class? I'm kinda lost trying to get a player's league information.

**Comment by MrXXIV** [comment]
I'll see what I can do. Considering I'm trying to make the most of using this repo for my League network, I'll try to help with the documentation. At first I got confused about the stat() method until I went to the API folder, so it's definitely needed.

**Comment by paquettg** [comment]
Any effort for documentation is welcomed @MrXXIV . I tried my best to document all the methods as best as I could but. some help on this tedious (but very important) task is appricated

**Comment by MrXXIV** [comment]
@paquettg no doubt. I'm grasping a huge understanding of the framework the more I use it and I'll be able to throw some extra details aside from what's already portioned in the current doc version.

I'm sure some people haven't noticed the **raw()** method yet and it's highly useful.

**Comment by PremadeGames** [comment]
Maybe you should use the repo wiki.. The Readme is kinda long.

**Comment by danijoo** [comment]
I agree with @PremadeGames. Unfortunatly I cant find time to work on the docs right now. Pull requests are welcome! :)
